templates/views/email-password-interaction/confirm.ejs

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html >
+  <head>
+    <meta charset="UTF-8">
+    <title>Authorize</title>
+    <style type="text/css">
+      <%- include('main.css', {}); %>
```

Comment:
Serve as asset?

Reply:
That would require configuring a `RouterHandler` to point to a new class like a `StaticServeHandler`, just to serve one css file.